packages/fuel-gauge/src/coverage-contract.test.ts

```diff
@@ -409,90 +407,6 @@ describe('Coverage Contract', () => {
     // #endregion
   });
 
-  it('should test sending input messages [1]', async () => {
```

Comment:
Instead of deleting this case, shouldn't we skip it? I believe we should bring this back once we support this feature again.

Reply:
I was talking with @Voxelot and @pixelcircuits, and Fuel does not support this feature. It was supposed to never work, but it was working before. This is why we are removing the tests